app/resources/drafts.py

```diff
@@ -101,22 +102,24 @@ def put(self, draft_id):
             return resp
 
     @staticmethod
-    def check_valid_draft(draft_id):
+    def check_valid_draft(draft_id, user_urn):
         """Check msg_id is that of a valid draft"""
```

Comment:
This is no longer just checking , its also getting so the function name and the code string are not correct . The name should reflect the function .  Maybe a get_draft and a separate is_valid_draft passing it the result of check draft ? . Or even a combined function with a better name ?